CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ find_package(Vulkan REQUIRED)
 
 add_custom_command(
 	OUTPUT "${CMAKE_BINARY_DIR}/Square.spv"
-	COMMAND $ENV{VK_SDK_PATH}/Bin/dxc -T cs_6_0 -E "Main" -spirv -fvk-use-dx-layout -fspv-target-env=vulkan1.1 -Fo "${CMAKE_BINARY_DIR}/Square.spv" "Square.hlsl"
```

Comment:
What if you don't have dxc available in the Path?

Reply:
Your version doesn't work for me since I don't have the SDK installed and dxc comes from a Debian package.
Mine doesn't work for you.
Obviously both should work.